$NetBSD: patch-aj,v 1.1 2003/01/05 03:31:26 dsainty Exp $

This patch changes the idle CPU wait time from 10ms to 100ms.  For some reason
the former saturates the CPU (on a Celeron 700 system), but with the latter
the program idles at near 0%.  This patch is probably not required for native
threads.

--- SrcUnix/Platform_Unix.cpp.orig	Sat Mar 30 01:11:19 2002
+++ SrcUnix/Platform_Unix.cpp	Sun Jan  5 15:01:21 2003
@@ -598,7 +598,8 @@
 	// Wait on an event instead of just calling Sleep(10) so that another
 	// thread can wake us up before our time.
 
-	omni_thread::sleep( 0, 10000 ); // 10k nanoseconds = 1/100 sec
+	// pkgsrc: updated 10k to 100k, makes a huge difference to CPU usage
+	omni_thread::sleep( 0, 100000 ); // 10k nanoseconds = 1/100 sec
 }
 
 
